dsl/camel-jbang/camel-jbang-core/src/main/resources/generator/quarkus-pom.ftl

```diff
@@ -0,0 +1,270 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
We should not include ASF license in the generated project code, as end users can use this for their own projects that is not ASF licensed